### DISCUSSION TITLE
大佬，程序运行完成了，但Neo4j没有生成，这还有个报错是啥原因呀

### DISCUSSION THREAD

**Discussion by TomTom18-eng** [tabby-sec/tabby#27] [Q&A]
![image](https://user-images.githubusercontent.com/50043324/182060492-f99e19a3-a11a-4ab5-b982-7ad91fcd561d.png)

![image](https://user-images.githubusercontent.com/50043324/182060536-7f892bc3-ea83-4052-8ec4-e2a4db695d34.png)

**Comment by wh1t3p1g** [comment]
内存溢出了哈
